Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/branch.py

```diff
@@ -76,7 +76,7 @@ def main(cls, args, repository, **kwargs):
             return 1
 
         if not args.issue:
-            args.issue = Terminal.input('Branch name: ')
+            args.issue = Terminal.input('Name of branch to create: ')
```

Comment:
Separate question, we don't really use this name as branch name, we prepend `eng/` to it, right?

Reply:
Unless the user prepends `eng/` themselves, yes.

@aproskuryakov had indicated he though "Branch name" wasn't specific enough, hence the change. Not sure what the right prompt here is.